test/components/dragBoxLayerTests.ts

```diff
@@ -106,6 +106,26 @@ describe("Interactive Components", () => {
         svg.remove();
       });
 
+      it("does not call callbacks after destroy() is called", () => {
+        let group = new Plottable.Components.Group([dbl]).renderTo(svg);
```

Comment:
Can we add a comment which explains this? It was not clear to me why is the group needed